bestofjs/bestofjs
# Migration backend / Add `build-static-api` task

## Goal

Another step toward the migration of the backend: we want to generate the static API (the JSON file consumed by the web app) from a build process running on Vercel.

This PR also schedules the `update-github-data` process to 6:30 JST, with a concurrency of `5` instead of `1`, to speed up the process.

## How to test

The following command should generate the file `projects.json` inside the `build` folder:

```
pnpm dotenv tsx apps/backend/src/cli.ts build-static-api -- --limit 5
```
